### PR TITLE
New Codon: UGO

### PR DESCRIPTION
**Pull request by Tux9001**
I've added a new codon that can be used to remove UGOs. This means it is possible to create "virus factories" out of cells.

### CODE REVIEW

**Comment by LegendaryHeart8** [comment]
This interests me, what exactly do you mean by virus factories? And, how are they any different from just making the viruses self-replicate?

**Comment by c00lum** [comment]
I believe what he is saying is instead of a cell making UGOs and then exploding with UGOs and killing itself in the process, a cell would be able produce a UGO, and then remove the UGO from within it's walls to avoid dying. This idea has alot of potential.

**Comment by Tux9001** [comment]
Sorry, just a bit of a joke that I forgot to remove.

**Comment by eciencs** [comment]
anyone else having trouble with recursion and this merge?

**Comment by TheRealEli310** [comment]
Can cary merge this? it sounds like a nice addition

**Comment by dkmk100** [comment]
I also think this should be added

File: virus/virus.pde [hunk 1]
```diff
@@ -405,6 +405,7 @@ void drawUI(){
   if(selectedCell != null){
     drawCellStats();
   }
+  //image(loadImage("nygi6iodek151.jpg"),0,0);
```
Comment (BLiu1) [inline virus/virus.pde:408]: Removing the two changes to this file would be nice

**Comment by carykh** [comment]
Ok I'll merge this

**Comment by BLiu1** [comment]
Wow! Cary's actually merging changes!

**Comment by Tux9001** [comment]
Wait, Cary actually merged MY pull request? LEGEND.

File: virus/virus.pde (hunk 1 above)
Comment (iamthewallrus) [reply virus/virus.pde:408]: agreed

Comment (iamthewallrus) [reply virus/virus.pde:408]: shit how do i cancel that

**Pull request merged**